docs: Reword the warning on environment variable usage (#555)
fleek-platform/website: src/content/docs/CLI/Functions/index.mdx

:::warn

> Your Fleek Function are stored on IPFS by default, which is publicly accessible, and will expose all of your code associated with the function including any environment variables which are bundled into your code.

Reviewer: ```suggestion
Your Fleek Functions are stored on IPFS by default, making them [publicly accessible](https://docs.ipfs.tech/concepts/privacy-and-encryption/). This will expose all your source code, including any environment variables bundled with it.
```

> Ensure you are not inadvertently exposing data you are interested in keeping private.
> If you need to upload sensitive data with your Fleek Function, you can make use of the `--private` flag when deploying and your Fleek Function will be stored on Fleek's private storage, which is not exposed to the public.

Reviewer: ```suggestion
To protect sensitive data and keep it private, you must deploy your Fleek Function with the `--private` flag. Click [here](https://fleek.xyz/docs/cli/functions/#making-fleek-functions-private--alpha) to learn more.
```
